packages/@okta/vuepress-theme-prose/components/MobileOnThisPage.vue

```diff
@@ -1,27 +1,39 @@
 <template>
   <div class="mobile-on-this-page" v-show="options.length > 0">
-    <h3>On This Page</h3>
-    <v-select :options="options" :searchable="false" :multiple="false" :clearable="false" v-on:input="inputChanged"></v-select>
+    <h3 class="mobile-header">On This Page</h3>
+    <v-select :options="options" v-model="selectedOption" :searchable="false" :multiple="false" :clearable="false" v-on:input="inputChanged">
+
+      <template #option="{label, isSubheading}">
+        <div class="dropdown-item" v-bind:class="{subheading: isSubheading}">
+          {{label}}
+        </div>
+      </template>
+    </v-select>
   </div>
 </template>
 
 <script>
   export default {
     name: 'MobileOnThisPage',
+    data: ()=>({ 
+     selectedOption: {}
+    }),
     computed: {
       options: function () {
         let optionsList = [];
         if( this.$page.headers ) {
+          console.log(this.$page.headers)
```

Comment:
@AndriiDovhopolov-okta  this console log statement should be removed.

@bretterer @swiftone can we lint for these?

Reply:
Thanks, @robertjd missed that one.